examples/land-cover-map/SI_LULC_pipeline.ipynb

```diff
@@ -2849,14 +2846,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# load sampled eopatches\n",
-    "eopatches = []\n",
-    "path_out_sampled = './eopatches_sampled'\n",
+    "# Load sampled eopatches\n",
+    "eopatch_samples = []\n",
```

Comment:
A better naming might be `sampled_eopatches` because this will be a list of EOPatches with spatially sampled pixels.